documentation/adr/0007_use_devise_for_authentication.md

```diff
@@ -0,0 +1,50 @@
+# Use Devise for authentication
+
+**Date: 2020-11-16**
+
+## Status
+
+**Decided**
+
+## Parties Involved
+
+ * Alex Bowen
+ * Cesidio Di Landa
+ * Christian Sutter
+ * Connor McQuillan
+ * David Mears
+ * Davide Dippolito
+ * Joe Hull
+
+## Context
+
+Teaching Vacancies will soon start to develop a new set of features which ultimately will allow job seekers to apply for jobs directly on the TVS website.
+
+This feature constitutes an important milestone for the service and will require job seekers users to create accounts to manage their job applications. Account functionality will permit teachers to initiate web sessions on TVS website as well as manage their account (password resets, email verification and account closing).
+
+## Options considered
+
+| Name      | Link                                     | Notes                                                                                    |
+| --------- | ---------------------------------------- | ---------------------------------------------------------------------------------------- |
+| Devise    | https://github.com/heartcombo/devise     | Widely adopted in the Ruby. Easily available documentation and features covering most TVS needs |
```

Comment:
`Widely adopted in the Ruby.` Did you mean the Ruby world?

Reply:
👍 Copy-paste error: addressed in 23a59f400c5a164cc0a9b70fcad932a0c02707e6